isaric/parsers/isaric_washington.toml

```diff
@@ -0,0 +1,614 @@
+    { field = "name_of_antiviral___7", values = { 1 = "Ritonavir" } },
+    { field = "name_of_antiviral___8", values = { 1 = "Other" } }
+]
+
+[visit.treatment_antibiotics]
+field = "medications___9"
+ref = "Y/N/NK"
+
+[visit.treatment_anticoagulation]
+combinedType = "any"
+fields = [
+    { field = "anticoagulation_category___2", ref = "Y/N/NK" },
+    { field = "anticoagulation_category___3", ref = "Y/N/NK" },
+    { field = "anticoagulation_category___4", ref = "Y/N/NK" },
+    { field = "anticoagulation_category___5", ref = "Y/N/NK" },
+    { field = "anticoagulation_category___6", ref = "Y/N/NK" },
+    { field = "anticoagulation_category___7", ref = "Y/N/NK" },
+    { field = "anticoagulation_category___8", ref = "Y/N/NK" },
+    { field = "anticoagulation_category___9", ref = "Y/N/NK" },
+    { field = "anticoagulation_category___10", ref = "Y/N/NK" },
+    { field = "anticoagulation_category___11", ref = "Y/N/NK" },
+    { field = "anticoagulation_category___12", ref = "Y/N/NK" },
+    { field = "anticoagulation_category___13", ref = "Y/N/NK" },
+    { field = "anticoagulation_category___14", ref = "Y/N/NK" },
+    { field = "anticoagulation_category___21", ref = "Y/N/NK" },
+    { field = "medications___8", ref = "Y/N/NK" }
+]
+
+
+[visit.treatment_inhaled_nitric_oxide]
+field = "treatment_events___2"
+ref = "Y/N/NK"
+
+[visit.treatment_noninvasive_ventilation]
+combinedType = "any"
+fields = [
+    { field = "oxygenation___3", ref = "Y/N/NK" },
+    { field = "oxygenation_methods_2___2", ref = "Y/N/NK" },
+]
+
+[visit.treatment_high_flow_nasal_cannula]
+combinedType = "any"
+fields = [
+    { field = "oxygenation___1", ref = "Y/N/NK" },
+    { field = "oxygenation_methods_2___3", ref = "Y/N/NK" },
+]
+[visit.treatment_steroids]
+field = "medications___16" 
+ref = "Y/N/NK"
+
+[visit.treatment_intravenous_fluids]
+field = "totak_iv_fluid"
+
+[visit.treatment_nsaid]
+field = "medications___16" 
+ref = "Y/N/NK"
+
+[visit.treatment_neuraminidase]
+field = "name_of_antiviral___4" 
+ref = "Y/N/NK"
+
+[visit.treatment_neuromuscular_blocking_agents]
+field = "nmb_given"
+description = "NeuroMuscular Blockers (NMB) Given?"
+values = { 0 = false, 1 = true, 2 = true, 3 = true, 4 = true, 5 = true, 6 = true, 7 = true }
+
+[visit.treatment_cpr] # not sure about this!
+field = "good_death___1"
+ref = "Y/N/NK"
+
+[visit.treatment_tocilizumab]
+field = "mis_medications___1"
+ref = "Y/N/NK"
+
+[visit.treatment_immunoglobulins]
+field = "blood_products___6"
+ref = "Y/N/NK"
+
+[visit.outcome]
+field = "hosp_status"
+description = "Discharged alive or dead"
+values = { 1 = "Deceased", 0 = "Alive" }
+
+[visit.date_outcome] # calculate: hospital length of stay (hospital_los) + the admission date - CHECK THIS!
+field = "admission_date"
+description = "outcome date calculated from admission and hospital los"
+apply = { function = "endDate", params = ["hospital_los"]}
+
+
+[[observation]] 
+name = "avpu"
+date = "date_filling_first_form"
```

Comment:
if this date is correct, it should read date = {field = "date_filling_first_form"}. But I think in this case, and a lot of the others, this is the wrong date - I'd suggest 
  date = { field = "ADMIT DATE", apply = { function = "endDate", params = [
    "hospital_day_core_e1",
  ] } }

Reply:
ok, nice, will try this